routers/menu.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require("express");
+const { meal, filling, topping, salsa } = require("../models");
+
+const router = new Router();
+
+router.get('/', async (req,res,next) => {
+
+	try{
+		const meals = await meal.findAll();
+		const fillings = await filling.findAll();
+		const toppings = await topping.findAll();
+		const salsas = await salsa.findAll();
+		const menu = { meals, fillings, toppings, salsas };
+		res.send(menu);
+		
+	}catch(e){
+		res.send(e);
+	};
+});
```

Comment:
I think your VScode might have the `tab-size` set to 8 spaces, check it out and put it down to 4